lib/phoenix/live_dashboard/telemetry_listener.ex

```diff
@@ -95,6 +111,9 @@ defmodule Phoenix.LiveDashboard.TelemetryListener do
       :telemetry.detach({__MODULE__, event, self()})
     end
 
+    # Clean up ETS table
+    :ets.delete(:dashboard_telemetry)
```

Comment:
There is no need to delete it, the table is automatically GCed if the process terminates.